components/Modal.js

```diff
@@ -1,10 +1,11 @@
 import styled from "styled-components";
 
-export default function Modal({ modalName, toggleModal }) {
+export default function Modal({ modalName, toggleModal, children }) {
```

Comment:
You would not usually name your prop modalName. `name` is sufficient since it is clear that it already belongs to the modal.

Reply:
Done.